Translated/Tasks/Daily/ed0610010101.csv

```diff
@@ -1,4 +1,4 @@
 Title#0,"""Central Aerio Vegetable Procurement"""
-Summary#0,"""Obtain vegetables by Gathering at Central Aerio.<br><br><yellow>※Target material:<br>\u3000マッタ・エアルハーヴ<c>"""
```

Comment:
Gathering is a game term, thus needs to be capitalized

Reply:
i see will revert those changes